Add minimal viable docs for addons (#11444)
readthedocs/readthedocs.org: docs/user/pull-requests.rst

       GitHub build status reporting

Pull request banner
    A pull request banner is shown at the top of documentation pages

>     to let readers know that this version isn't the main version for the project.

Reviewer: ```suggestion
    to let readers know they aren't viewing an active version of the project.
```

>     .. note:: Warning banners are available only for :doc:`Sphinx projects </intro/getting-started-with-sphinx>` or projects using :doc:`Read the Docs Addons </addons>`.

Reviewer: ```suggestion
    .. note:: Warning banners are available only for projects using :doc:`Read the Docs Addons </addons>`.
```

I think this is probably a better place for now -- and once we fully ship them, we can remove this.
